pkg/controller/remoteingress/remoteingress_controller.go

```diff
@@ -313,26 +318,27 @@ func (r *ReconcileRemoteClusterIngress) syncSyncSet(rContext *reconcileContext,
 // createSecret returns the secret that needs to be synced to the remote cluster
 // to satisfy any ingressController object that depends on the secret.
 func createSecret(rContext *reconcileContext, certBundle hivev1.CertificateBundleSpec) (*corev1.Secret, error) {
-	secretData, ok := rContext.certBundleSecrets[certBundle.Name]
-	if !ok {
-		// this really shouldn't happen as earlier in Reconcile()
-		// we make sure all the secrets are available
-		return nil, fmt.Errorf("did not find secret data for certBundle: %v", certBundle.Name)
-	}
+	for _, secret := range rContext.certBundleSecrets {
+		if secret.Name == certBundle.SecretRef.Name {
+			newSecret := corev1.Secret{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Secret",
+					APIVersion: corev1.SchemeGroupVersion.String(),
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      secretNameForCertificateBundle(certBundle.Name, rContext.clusterDeployment),
+					Namespace: remoteIngressConrollerSecretsNamespace,
+				},
+				Data: secret.Data,
+			}
```

Comment:
@joelddiaz you should copy over the secret type at the very least.
I'll leave it up to you if you think you should also copy annotations and labels

Reply:
done. now i deepcopy the original secret and just update the name and namespace where it needs to land

Reply:
@joelddiaz that sounds fine, though it'd be good to test it. There's a few system-generated in object meta that don't apply to a new object (UID, selfLink, resourceVersion). Which is why I only copy things explicitly --> https://github.com/openshift/hive/pull/310/files#diff-b945d6ca702f17e881579fd8805078cfR337